transport/rest/server.go

```diff
@@ -43,7 +43,10 @@ func (h *Handler) Listen() {
 	// LH side
 	lhApi := h.router.PathPrefix("/lh/v1").Subrouter()
 	lhApi.HandleFunc("/quote", h.amountOut).Methods("POST")
-	lhApi.HandleFunc("/approve_orders", h.approveOrders).Methods("GET")
+	lhApi.HandleFunc("/confirm_auction/{auctionId}", h.confirmAuction).Methods("GET")
```

Comment:
None of these should be GET requests right? DELETE or POST/PUT?

Reply:
yeah lets change that,